tool/resources/org/antlr/v4/tool/templates/codegen/Java/Java.stg

```diff
@@ -715,7 +725,7 @@ ArgAction(a, chunks) ::= "<chunks>"
 
 SemPred(p, chunks, failChunks) ::= <<
 setState(<p.stateNumber>);
-if (!(<chunks>)) throw new FailedPredicateException(this, <p.predicate><if(failChunks)>, <failChunks><elseif(p.msg)>, <p.msg><endif>);
+if (!(<chunks>)) throw createFailedPredicateException(this, <p.predicate><if(failChunks)>, <failChunks><elseif(p.msg)>, <p.msg><endif>);
```

Comment:
❔ Since `createFailedPredicateException` is an instance method, do we need to pass it as an argument here?